plat/xen/arm/gnttab.c

```diff
@@ -34,6 +34,8 @@
 #include <xen-arm/os.h>
 #include <xen-arm/mm.h>
 
+extern lpae_t fixmap_pgtable[512];
```

Comment:
Please capitalize at the start of the sentence (`That's why..`)

Reply:
fixed